ee/desktop/runtime/runner_darwin.go

```diff
@@ -47,7 +47,7 @@ func (r *DesktopUsersProcessesRunner) runConsoleUserDesktop() error {
 		return fmt.Errorf("determining executable path: %w", err)
 	}
 
-	proc, err := runAsUser(uid, executablePath, "desktop")
+	proc, err := runAsUser(uid, executablePath, "desktop", fmt.Sprintf("--hostname=%s", r.hostname))
```

Comment:
```suggestion
	proc, err := runAsUser(uid, executablePath, "desktop", "--hostname", r.hostname)
```

I think this should work